### PR TITLE
Add page for Hackathon2024

### PR DESCRIPTION
**Pull request by 49Indium**
Add Hackathon2024 to [the competitions page](https://uqcs.org/competitions/). Blurb stole shamelessly from the announcements and merge with last years prose.

### CODE REVIEW

**Review by nicklambourne — APPROVED** [review]
LGTM

File: content/competitions/_index.html [hunk 1]
```diff
@@ -15,10 +15,14 @@ <h1 class="title is-3">UQCS Competitions</h1>
                 We usually run two major competitive events each year: the UQCS CodeJam in Semester 1, and the UQCS Hackathon in Semester 2.
             </p>
 
-            <h4><a href="hackathon-2023/">UQCS Hackathon 2023</a> (25th &ndash; 27th August 2023)</h4>
-
+            <h4><a href="hackathon-2024/">UQCS Hackathon 2024</a> (16th &ndash; 18th August 2024)</h4>
+            
             <h2>Archive</h2>
 
+            <h4>UQCS CodeJam 2024 (13th April 2023)</h4>
```
Comment (katrinafyi) [inline content/competitions/_index.html:22]:
```suggestion
            <h4>UQCS CodeJam 2024 (13th April 2024)</h4>
```

File: content/competitions/hackathon-2024.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+title: "Hackathon 2024"
+date: 2024-07-26T22:14:00+10:00
+draft: false
+---
+
+<div class="section columns is-centered">
+    <section class="column is-6">
+
+        <div class="block">
+            <a href="https://fb.me/e/7qY5282Ut">
```
Comment (katrinafyi) [inline content/competitions/hackathon-2024.html:11]: link doesn't seem to work on my phone? possibly a me issue, feel free to disregard. ( this does: https://facebook.com/events/s/uqcs-hackathon-2024/497939952616194/ )

File: content/competitions/hackathon-2024.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                <a href="https://events.humanitix.com/uqcs-hackathon-2024" class="button is-link uqcs-blue">Get your tickets now!</a>
+                <a href="https://fb.me/e/7qY5282Ut" class="button">Facebook event</a>
+            </div>
+
+			<p>
+                Bring your creativity, friends, and coding skills for a weekend filled with innovation and excitement! Get ready for UQCS's biggest event of the year: Hackathon! The goal of hackathon is to build something amazing over the weekend, whether it's games, websites, apps, hardware, or more. Free meals are provided throughout the event, and you can stay overnight at UQ to code uninterrupted through the night.
+			</p>
+
+			<p>
+                Everyone is welcome, regardless of your year or experience level, and its not necessary to have a team beforehand. If you’re new to hackathons, don’t worry—we will be hosting a talk before the hackathon weekend to clear up any confusion and give you tips and tricks to help you enjoy your hackathon experience! For inspiration, check out some past projects in <a href="/showcase/">our showcase</a>.
```
Comment (katrinafyi) [inline content/competitions/hackathon-2024.html:31]:
```suggestion
                Everyone is welcome, regardless of your year or experience level, and it's not necessary to have a team beforehand. If you’re new to hackathons, don’t worry—we will be hosting a talk before the hackathon weekend to clear up any confusion and give you tips and tricks to help you enjoy your hackathon experience! For inspiration, check out some past projects in <a href="/showcase/">our showcase</a>. 
```

**Review by katrinafyi — COMMENTED** [review]
some buttons for you to click.

File: content/competitions/hackathon-2024.html (hunk 2 above)
Comment (49Indium) [reply content/competitions/hackathon-2024.html:11]: Seems to work on desktop and my phone, and follows same pattern as last year. I'll keep as is for now.

**Pull request merged**